siroc.config.ts

```diff
@@ -0,0 +1,14 @@
+import { PackageOptions } from 'siroc'
+import copyPlugin from 'rollup-plugin-copy'
+
+const config: PackageOptions = {
+  rollup: {
+    plugins: [
+      copyPlugin({
+        targets: [{ src: 'src/templates/*', dest: 'lib/templates' }]
```

Comment:
We may alternatively use relative path to templates and keep it on top level to avoid copy. (both src/ and dist/ are in a same level)

Reply:
:+1: